server/lib/src/idm/ldap.rs

```diff
@@ -240,6 +240,7 @@ impl LdapServer {
             let mut all_op_attrs = false;
 
             // TODO #67: limit the number of attributes here!
+            // Is this still needed?
```

Comment:
Yeah, probably, we should have an issue for it. It would be pretty easy to add IMO.